docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 5
+title: Tokens Overview
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar: 1) assets issued by Stellar accounts (Stellar assets can have a [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation), and 2) as a [custom token](./token-interface.mdx) created by a smart contract.
```

Comment:
I think it might be easier to read if these 1 and 2 points are on new lines in a number list form.